src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -259,6 +264,28 @@ debug_configuration::debug_configuration()
         exit(0);
     }
 
+    if (dump_prof_data_iter_str.length() > 0) {
+        dump_prof_data_iter_str = " " + dump_prof_data_iter_str + " ";
+        std::istringstream iss(dump_prof_data_iter_str);
+        char dot;
+        int start, end;
+        bool is_valid_range = false;
+        if (iss >> start >> dot >> dot >> end) {
+            if (start < end) {
```

Comment:
I think it would be good for "start <= end" case too, because we may want to check just one specific iteration.

Reply:
Also what if the end count is not specified? e.g., It would be good to support open ended range like : "100,-1" to profile from 100 to the end of the execution

Reply:
Update to support both one specific iteration and open ended range